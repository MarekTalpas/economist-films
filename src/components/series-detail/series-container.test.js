// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import SeriesContainer from './series-container';

const mockData = {
  description: 'aaa',
  additional_assets: [
    {
      file: {
        url: 'xxx',
      },
    },
    {
      file: {
        url: 'yyy',
      },
    },
    {
      file: {
        url: 'zzz',
      },
    },
  ],
};

describe('SeriesContainer', () => {
  test('renders correctly with data', () => {
    const tree : string = renderer.create(
      <SeriesContainer
        series={mockData}
        params={{ id: 5 }}
      />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders correctly without data', () => {
    const tree : string = renderer.create(
      <SeriesContainer
        params={{ id: 5 }}
      />,
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
