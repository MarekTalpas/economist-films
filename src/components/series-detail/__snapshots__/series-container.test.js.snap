// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SeriesContainer renders correctly with data 1`] = `null`;

exports[`SeriesContainer renders correctly without data 1`] = `null`;
